{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/exception\/ciphertextsize(value:)"]}],"metadata":{"externalID":"s:10SwiftKyber9ExceptionO14cipherTextSizeyACSi_tcACmF","symbolKind":"case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cipherTextSize"},{"kind":"text","text":"("},{"kind":"externalParam","text":"value"},{"kind":"text","text":": "},{"preciseIdentifier":"s:Si","kind":"typeIdentifier","text":"Int"},{"kind":"text","text":")"}],"roleHeading":"Case","title":"Exception.cipherTextSize(value:)","modules":[{"name":"SwiftKyber"}],"role":"symbol"},"schemaVersion":{"major":0,"minor":3,"patch":0},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/cipherTextSize(value:)"},"hierarchy":{"paths":[["doc:\/\/SwiftKyber\/documentation\/SwiftKyber","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception"]]},"sections":[],"abstract":[{"type":"text","text":"Wrong cipher text size"}],"kind":"symbol","primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cipherTextSize"},{"kind":"text","text":"("},{"kind":"externalParam","text":"value"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Int","preciseIdentifier":"s:Si"},{"kind":"text","text":")"}]}],"kind":"declarations"}],"references":{"doc://SwiftKyber/documentation/SwiftKyber/Exception":{"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Exception"}],"type":"topic","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception","role":"symbol","title":"Exception","url":"\/documentation\/swiftkyber\/exception","abstract":[{"text":"The Kyber exceptions","type":"text"}],"kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"Exception"}]},"doc://SwiftKyber/documentation/SwiftKyber":{"title":"SwiftKyber","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber","role":"collection","kind":"symbol","type":"topic","url":"\/documentation\/swiftkyber","abstract":[{"text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard","type":"text"}]},"doc://SwiftKyber/documentation/SwiftKyber/Exception/cipherTextSize(value:)":{"url":"\/documentation\/swiftkyber\/exception\/ciphertextsize(value:)","title":"Exception.cipherTextSize(value:)","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/cipherTextSize(value:)","kind":"symbol","type":"topic","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cipherTextSize"},{"kind":"text","text":"("},{"kind":"externalParam","text":"value"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:Si","text":"Int"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"Wrong cipher text size"}]}}}